### ISSUE TITLE
Things to do when we move to wlroots 0.17

### ISSUE THREAD

**Issue by Consolatis** [labwc/labwc#727]
Nothing immediate, just want to preserve this list somewhere

Scene:
- [ ] wlr_scene got a drag icon helper which is exactly the same as what we have in `src/dnd.c` currently
- [ ] use wlr_scene_buffer `outputs_update` event for the autoscaling buffer because we get all outputs instead of just primary and newly entered

**Comment by jlindgren90** [comment]
We can perhaps replace 753+785+788 with #790.

**Comment by Consolatis** [comment]
Feel free to edit the original message

**Comment by fungilife** [comment]
Are there distros that still build labwc on wlroots 0.17?

**Comment by stefonarch** [comment]
![immagine](https://github.com/user-attachments/assets/8d40b3ac-be28-4ca8-81a1-858e04f7bd80)

I can't build latest labwc on debian testing/experimental.

**Comment by Consolatis** [comment]
> Are there distros that still build labwc on wlroots 0.17?

Yes: https://repology.org/badge/vertical-allrepos/labwc.svg?exclude_unsupported=1
Note that labwc 0.7.x is always built with wlroots 0.17.x and labwc 0.8.x is always built with wlroots 0.18.x. The labwc 0.7.x branch is in maintenance mode so only critical bugfixes are accepted for that branch and all development takes place in the 0.8.x branch.

> I can't build latest labwc on debian testing/experimental.

We do build on Debian Testing as part of the CI: https://github.com/labwc/labwc/blob/master/.github/workflows/build.yml#L76

**Comment by stefonarch** [comment]
No idea, I always get 
```
ependency wlroots-0.18 from subproject subprojects/wlroots found: NO found 0.16.2 but need: '>=0.18.1', '<0.19.0'

meson.build:52:10: ERROR: Dependency 'wlroots-0.18' is required but not found.
```

**Comment by Consolatis** [comment]
Looks like your subproject is slightly too old.
```
cd subprojects/wlroots
git pull
git checkout 0.18
cd ../..
```
should fix that.

**Comment by stefonarch** [comment]
Thanks, indeed.
